### ISSUE TITLE
erroneous call to _to_api_data

### ISSUE THREAD

**Issue by thomas-schulze-entwicklerschmiede** [janscas/py-o365#23]
When working with addressbooks, I hit a problem with pyo365/address_book.py:124:

cf. pyo365/address_book.py : 112ff 
```python
       data = {
            'displayName': self.display_name,
            'givenName': self.name,
            'surname': self.surname,
            'title': self.title,
            'jobTitle': self.job_title,
            'companyName': self.company_name,
            'department': self.department,
            'officeLocation': self.office_location,
            'businessPhones': self.business_phones,
            'mobilePhone': self.mobile_phone,
            'homePhones': self.home_phones,
            'emailAddresses': self.emails.to_api_data(),
            'businessAddress': self.business_addresses,
            'homesAddress': self.home_addresses,
            'otherAddress': self.other_addresses,
            'categories': self.categories}
        return data
```

This gives the error:
> AttributeError: 'Recipients' object has no attribute 'to_api_data'

I changed pyo365/address_book.py to:
```python
       data = {
            'displayName': self.display_name,
            'givenName': self.name,
            'surname': self.surname,
            'title': self.title,
            'jobTitle': self.job_title,
            'companyName': self.company_name,
            'department': self.department,
            'officeLocation': self.office_location,
            'businessPhones': self.business_phones,
            'mobilePhone': self.mobile_phone,
            'homePhones': self.home_phones,
            'emailAddresses': self.emails,
            'businessAddress': self.business_addresses,
            'homesAddress': self.home_addresses,
            'otherAddress': self.other_addresses,
            'categories': self.categories}
        return data
```

Now this seems to work!?

**Comment by thomas-schulze-entwicklerschmiede** [comment]
small update...

I changed the code in pyo365/address_book.py back to original state,
since my change broke the implementation.

I then tried the example in the documentation:
```Python
address_book = account.address_book()
new_contact = address_book.new_contact()
new_contact.name = 'George Best'
new_contact.job_title = 'football player'
new_contact.emails.add('george@best.com')
new_contact.save()  # saved on the cloud
```

Even that examples failes with 

> AttributeError: 'Recipients' object has no attribute 'to_api_data'

What am I doing wrong here?

Best regards

**Comment by alejcas** [comment]
It was a bug. I have just fixed it.

**Comment by thomas-schulze-entwicklerschmiede** [comment]
Hi Alejandro,

strangely now installing via pi + git fails - it appears, it doesn't install correctly!?

ModuleNotFoundError: No module named 'pyo365.__version__'

Kind regards


—
Ihr Ansprechpartner:
Dipl.-Ing.
Thomas Schulze
Chief Executive Officer
Tel.: +49 9391 6905
thomas.schulze@entwicklerschmiede.com<mailto:thomas.schulze@entwicklerschmiede.com>

Es unterstützt Sie
Die Entwicklerschmiede GmbH & Co. KG
Würzburger Strasse 5
97828 Marktheidenfeld

www.entwicklerschmiede.com
info@entwicklerschmiede.com

USt-IdNr.: DE305944872

CONFIDENTIALITY NOTICE
This email is for the use of the intended recipient only. It may contain information that is legally privileged or confidential.
If you are not the intended recipient, any disclosure, distribution or copying of this email is strictly prohibited and may be unlawful.
If received in error, please reply to the sender confirming this, then delete the email.
Der Inhalt dieser E-Mail ist ausschliesslich für den bezeichneten Adressaten bestimmt.
Wenn Sie nicht der vorgesehene Adressat dieser E-Mail oder dessen Vertreter sein sollten, so benachrichtigen Sie uns bitte und beachten Sie,
dass jede Form der Kenntnisnahme, Veröffentlichung, Vervielfältigung oder Weitergabe des Inhalts dieser E-Mail unzulässig ist.

Am 26.11.2018 um 10:11 schrieb Alejandro Casanovas <notifications@github.com<mailto:notifications@github.com>>:


It was a bug. I have just fixed it.

**Comment by alejcas** [comment]
Yep! there was another bug... Fixed in b10bf96.

Try now

**Comment by alejcas** [comment]
Also, note this project was merged into the original one [(O365)](https://github.com/O365/python-o365).

The development of this package will continue on O365.

You should try pip install O365 instead.
